tidb-cloud/configure-maintenance-window.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Configure Maintenance Window
+summary: Learn how to configure maintenance window for your cluster.
+---
+
+# Configure Maintenance Window
+
+A maintenance window is a designated timeframe during which planned maintenance activities, such as operating system updates, security patches, and infrastructure upgrades, are performed automatically to ensure the reliability, security, and performance of the TiDB Cloud service.
+
+During a maintenance window, the maintenance is executed on TiDB clusters one by one so the overall impact is minimal. Although there might be temporary connection disruptions or QPS fluctuations, the clusters remain available, and the existing data import, backup, restore, migration, and replication tasks can still run normally.
+
+By configuring the maintenance window, you can easily schedule and manage maintenance activities to minimize the maintenance impact. For example, you can set the start time of the maintenance window to avoid peak hours of your application workloads.
+
+> **Note:**
+>
+> - The maintenance window feature is only available for [Dedicated Tier clusters](/tidb-cloud/select-cluster-tier.md#dedicated-tier).
+> - If a TiDB Dedicated Tier cluster has only one TiDB node, it might be unavailable for a few minutes during maintenance.
```

Comment:
Basically LGTM, but there's one point. Although the current description is fine, we think it has a significant impact. We actually want to remove this from the technical aspect to the description, and we are still discussing it internally. Ultimately, we don't want to mention that there are "a few minutes unavailable". Let's wait for our conclusion before we publish it. Otherwise, it would be better to remove the entire sentence before publishing.